docs/cody/faq.mdx

```diff
@@ -25,6 +25,10 @@ Yes, Cody is compatible with self-hosted Sourcegraph instances. However, there a
 - For certain repositories, Cody may utilize embeddings, which involves sending repository data to another third-party service like OpenAI
 - To use Cody effectively, your self-hosted Sourcegraph instance must have internet access for these interactions with external services
 
+### Is Cody licensed for private code, and does it allow GPL-licensed code?
+
+There are no checks or exclusions for Cody PLG (VS Code, JetBrains, Neovim) for private and GPL-licensed code. We are subject to whatever the LLMs are trained on. However, using [StarCoder for autocomplete](/cody/clients/enable-cody-enterprise#use-starcoder-for-autocomplete) in PLG is trained on permissively licensed code.
```

Comment:
"However, using [StarCoder for autocomplete](/cody/clients/enable-cody-enterprise#use-starcoder-for-autocomplete) in PLG is trained on permissively licensed code." 

The sentence feels off to me. What do you think about:

"However, Cody can be used with StarCoder for autocomplete which is trained only on permissively licensed code. "

We can then link to some external doc on how StarCoder is trained on permissive code. Thoughts?

Reply:
@aramaraju external as far as some official link is available I guess we can link to it.

Reply:
Thoughts on this one:
https://huggingface.co/bigcode/starcoder#%23model-summary 
In their Attribution section, they mention permissive code.